zad1/main.rb

```diff
@@ -1,13 +1,23 @@
 def sort_letters(string)
-
+p= string.chars.sort.join
+    return p
 end
 
 def count_vowels(string)
-
+    sum=0
+    (0...string.size).each do |i|
```

Comment:
Można by się pozbyć zmiennej result, przez wykorzystanie metody `chars` której użyłeś powyżej i na zwróconym arrayu użyć metody [count](https://ruby-doc.org/core-2.5.3/Enumerable.html#method-i-count). Można zrobić to oczywiście jeszcze prościej używając metody `count`, która występuje analogicznie w klasie [String](https://ruby-doc.org/core-2.2.0/String.html#method-i-count)